src/devices/Ft4222/Ft4222Common.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Text;
+using Iot.Device.FtCommon;
+
+namespace Iot.Device.Ft4222
+{
+    /// <summary>
+    /// Common static functions for the FT4222
+    /// </summary>
+    public class Ft4222Common
```

Comment:
honestly I'd just recommend people to always use FtCommon and either remove this or make it internal (depending if it's used anywhere internally)